recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -40,6 +40,14 @@ set "PATH=%PATH:C:\Strawberry\perl\bin;=%"
 set "PATH=%PATH:C:\Strawberry\perl\site\bin;=%"
 set "PATH=%PATH:c:\tools\php;=%"
 
+if "%CI%" == "azure" (
```

Comment:
I would have thought that it couldn't hurt to run this on AppVeyor either? Since it wouldn't produce an error anyway?

Reply:
Sure. we should check that `CI` env variable is set though.